### PR TITLE
allow prometheus-exporter to export metrics on workload level

### PR DESCRIPTION
**Pull request by hebestreit**
## Overview

With this change it's possible to set extra environment variables for the prometheus-exporter. This is desired to set `ENABLE_WORKLOAD_METRICS=true`. Additionally I've updated the image tag to the latest version.

## Additional Information

The logic has been implemented in this PR:
https://github.com/kubescape/prometheus-exporter/pull/24

~~Don't be confused by the version reduction associated with CI creating negative versions:~~
https://github.com/kubescape/prometheus-exporter/issues/26

The Docker tag `v0.2.0` has been created manually to avoid the above problem.
https://quay.io/repository/kubescape/prometheus-exporter?tab=tags

## How to Test

Create a `values.yaml` with minimal config:

```yaml
capabilities:
  prometheusExporter: enable
prometheusExporter:
  env:
    - name: ENABLE_WORKLOAD_METRICS
      value: "true"
```

Render the helm template and and check the occurrence:
```
helm template kubescape-operator kubescape-operator -f values.yaml | grep -5 "ENABLE_WORKLOAD_METRICS"
                  resource: limits.cpu
            - name: KS_LOGGER_LEVEL
              value: "info"
            - name: KS_LOGGER_NAME
              value: "zap"
            - name: ENABLE_WORKLOAD_METRICS
              value: "true"
          volumeMounts:
            - name: ks-cloud-config
              mountPath: /etc/config
              readOnly: true

```

### CODE REVIEW

File: charts/kubescape-operator/values.yaml [hunk 1]
```diff
@@ -772,6 +772,11 @@ prometheusExporter:
     targetPort: 8080
     protocol: TCP
 
+  env: []
```
Comment (matthyx) [inline charts/kubescape-operator/values.yaml:775]: I would prefer to have a variable enableWorkloadMetrics that would add ENABLE_WORKLOAD_METRICS in the deployment

**Review by matthyx — CHANGES_REQUESTED** [review]
please also enable the new feature in `snapshot_test.yaml` ("all capabilities" scenario) and update unittests

File: charts/kubescape-operator/values.yaml (hunk 1 above)
Comment (hebestreit) [reply charts/kubescape-operator/values.yaml]: Sounds good to me too. Please take a look and let me know if I've missed anything.

**Review by matthyx — APPROVED** [review]
awesome, thanks @hebestreit !
I'm not sure we'll release before Christmas, but definitely in the beginning of 2025

**Pull request merged**